src/app/pages/join-session/join-session.component.html

```diff
@@ -10,7 +10,14 @@
         </mat-form-field>
         <mat-form-field appearance="outline" class="form-field">
           <mat-label>Name</mat-label>
-          <input matInput class="name"  formControlName="nameControl">
+          <input matInput class="name" formControlName="nameControl">
+        </mat-form-field>
+        <mat-form-field appearance="outline" class="form-field">
+          <mat-label>Nest Barcode</mat-label>
```

Comment:
"Barcode" is likely not a good description for this public facing interface. I would suggest something like "Nest marker ID" or "Nest marker number." I think we will be referring to the barcodes as "markers"